src/TodoGroup/TodoGroup.js

```diff
@@ -1,18 +1,31 @@
-import React from "react";
+import React, {useState} from "react";
 import TodoItem from "../TodoItem/TodoItem";
+import AddTodo from "../AddTodo/AddTodo";
 import './styles.css'
 
-function TodoGroup({todoList,color,created,title}){
+function TodoGroup({todoList,color,created,title, id, addTodo}){
+
+    const [isDone, setIsDone] = useState(false)
+
+    const handleChane=(done)=>{
+        setIsDone(!done)
+    }
 
     const todoItemsList = todoList.map((item, index)=>{
 
         if (item.hasOwnProperty('filtered')&& item.filtered !== false) {
             return <TodoItem key={index}
                              title={item.title}
-                             done={item.done}
+                             done={isDone}
+                             handleChange={handleChane}
+
             />} return null
     })
 
+
```

Comment:
There should be only one line between todoItemsList and return code

Reply:
I have only only one line

Reply:
return(...) is on 29. todoList.map end function is on 23. So there is 6 lines between the code, correct?